.github/workflows/export-notion-blocks-and-commit.yml

```diff
@@ -1,6 +1,7 @@
 name: 'Export Notion Blocks and Commit to Git'
 
 on:
+  workflow_dispatch:
```

Comment:
Can this be moved to the bottom of the `on:` section?

Reply:
Done!